src/libostree/ostree-sepolicy.c

```diff
@@ -257,6 +260,18 @@ initable_init (GInitable     *initable,
         freecon (con);
       }
 
+      {
+        g_autofree guchar *csum_raw = NULL;
+        if (!ostree_checksum_file (etc_selinux_dir, OSTREE_OBJECT_TYPE_FILE,
```

Comment:
Right, it just did a checksum of the directory metadata, which is of course not what we want. Should have read more carefully.

I initially just thought checksumming the whole /etc/selinux dir was a safer & easier way to handle this, but it seems like that may be overkill and the only file that actually matters is the final compiled policy file, e.g. `policy.29`.

Should we just use `dirname(selinux_binary_policy_path())` and then checksum all the files under there (though there should only be one for the matching kernel it seems) ? Not sure yet if the path it returns respects `selinux_set_policy_root()` or if we'd have to build the full dirname ourselves.